### ISSUE TITLE
Not auto redirect to correct sitemap URL

### ISSUE THREAD

**Issue by tdtgit** [tdtgit/sitemap-warmer#4]
**Describe the bug**
When using CLI with a domain only, ex. `warmup domain.com`, the Warmer did not redirect to correct sitemap URL: https://domain.com/sitemap_index.xml

**To Reproduce**
`warmup domain.com/sitemap404.xml`
or
`warmup notexistsdomain.com`

**Expected behavior**
Automatically redirect to `https://domain.com/sitemap_index.xml`

**Logs**
```2021-01-10T03:53:26.384Z [INFO] main: 📬 Getting sitemap from http://xn--version-t16c/sitemap.xml
/usr/local/lib/node_modules/datuan-sitemap-warmer/node_modules/sitemap-xml-parser/lib/sitemap.js:106
resolve(body.toString());
^

TypeError: Cannot read property ‘toString’ of undefined
at Request._callback (/usr/local/lib/node_modules/datuan-sitemap-warmer/node_modules/sitemap-xml-parser/lib/sitemap.js:106:34)
at self.callback (/usr/local/lib/node_modules/datuan-sitemap-warmer/node_modules/request/request.js:185:22)
at Request.emit (events.js:198:13)
at Request.onRequestError (/usr/local/lib/node_modules/datuan-sitemap-warmer/node_modules/request/request.js:877:8)
at ClientRequest.emit (events.js:198:13)
at Socket.socketErrorListener (_http_client.js:401:9)
at Socket.emit (events.js:198:13)
at emitErrorNT (internal/streams/destroy.js:91:8)
at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
at process._tickCallback (internal/process/next_tick.js:63:19)```

**Comment by tdtgit** [comment]
The issue due to https://github.com/shinkawax/sitemap-xml-parser lib did not catch the error (`resolve(body.toString());`) when the domain is not exist or sitemap URL is 404.

I think I will do a pre-check before sending to parser, then rewrite it later if need to. It's 2 year old code already.

**Comment by tdtgit** [comment]
Fixed in 7c873565f65762a34fc4eb534721b75fe7d23eb1
